### ISSUE TITLE
Actor On Use Macros prints an error.

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#828]
In GitLab by @brimconsako on May 1, 2022, 14:39

**Foundry Version: 9.266 AND 269**

**D&D5e 1.6.0 AND 1.5.7**

**Midi-QOL: 0.9.50**

**Libwrapper 1.12.4.0**

With only Midi-QOL and its dependancies enabled clicking on `Actor On Use Macros` prints this error and does not render the interface.
```js
foundry.js:319 SyntaxError: Unexpected token u in JSON at position 0
[Detected 1 package: midi-qol]
    at JSON.parse (<anonymous>)
    at duplicate (helpers.mjs:118)
    at ActorOnUseMacrosConfig.getData (ActorOnUseMacroConfig.js:22)
    at async ActorOnUseMacrosConfig._render (foundry.js:3062)
    at async ActorOnUseMacrosConfig._render (foundry.js:3764)
```

Here is the full error.

![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/2fe24ab101ac5127d4526bc31a05d2ff7a237bb0d0cdad1ea733c65c0067e923/image.png)

**Comment by tposney** [comment]
Can you export the actor that is having the problem? I'm guessing that for some reason the onUse macro field is undefined (the u), but it should not be. Would be nice to have a test case

**Comment by tposney** [comment]
In GitLab by @brimconsako on May 2, 2022, 01:42

[fvtt-Actor-beiro-_half-elf-bard_.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/c91f4ad1b65c27e0e84e143a24cf51b6a879310159d15ee3b06baebc1b3f1e6b/fvtt-Actor-beiro-_half-elf-bard_.json)

Sure, Here you go. I was testing it with one of the sample characters. But it occurs with every single actor I have both on a local and remote (Local is 1.6, Remote is 1.5.7)

**Comment by tposney** [comment]
Thanks - fix in 0.9.51
